packages/@aws-cdk/aws-kms/test/test.key.ts

```diff
@@ -475,51 +477,87 @@ export = {
     test.done();
   },
 
-  'import/export can be used to bring in an existing key'(test: Test) {
-    const stack2 = new Stack();
-    const myKeyImported = Key.fromKeyArn(stack2, 'MyKeyImported', 'arn:of:key');
+  'keyId resolves to a Ref'(test: Test) {
+    const stack = new Stack();
+    const key = new Key(stack, 'MyKey');
 
-    // addAlias can be called on imported keys.
-    myKeyImported.addAlias('alias/hello');
+    new CfnOutput(stack, 'Out', {
+      value: key.keyId,
+    });
 
-    expect(stack2).toMatch({
-      Resources: {
-        MyKeyImportedAliasB1C5269F: {
-          Type: "AWS::KMS::Alias",
```

Comment:
missing tests around the new behavior of grant

Reply:
It was tested in the pipeline tests, but you're right, it should be also tested explicitly. Done.